Allow to store directly into a temporary file

manuelnaranjo opened the pull request:
On Android we now allow to on-demand store into a temporary file. There's a chance for a race condition, which I'm not going to take into account, which is the case where multiple images are been taken and the file is not been read on time. But it's a trade-off we need to accept.

Documentation has been updated, also reflecting the fact that only 1 image is possible to take at the time.

No extra permissions are required as image is stored inside the app cache.

westonganger commented:
Looks good, however I dont see any documentation for`fetchLocal`. Also maybe this method name could be improved as well, but without the documentation I cant make that call.

manuelnaranjo commented:
Right I will document the method and maybe add and example or remove it all
together

El mar., 13 de mar. de 2018 22:32, Weston Ganger <notifications@github.com>
escribió:

> Looks good, however I dont see any documentation forfetchLocal. Also
> maybe this method name could be improved as well, but without the
> documentation I cant make that call.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cordova-plugin-camera-preview/cordova-plugin-camera-preview/pull/441#issuecomment-372825876>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAhUW6CngV8HfoB6ZgTLAp11qgRnSS--ks5teDr3gaJpZM4SpRsX>
> .
>

westonganger commented:
Any update on this?

manuelnaranjo commented:
Sorry I didn't have the time, but we released our app and already got 1k+
installs, not sure how many photos, but I got no big reports regarding
photos 😃

I'll update it once I have time maybe next week

El mié., 28 de mar. de 2018 23:58, Weston Ganger <notifications@github.com>
escribió:

> Any update on this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cordova-plugin-camera-preview/cordova-plugin-camera-preview/pull/441#issuecomment-377051336>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAhUW-ioXQ1yPTRRgeYqwhm26IGg6gdNks5tjAeJgaJpZM4SpRsX>
> .
>

westonganger commented:
Any time to work on completing this yet? Im sure some people would appreciate this.

manuelnaranjo commented:
I moved out of the project using it and totally forgot, I'll try to find
some time at my work this week to get it updated

El mié., 9 de may. de 2018 19:04, Weston Ganger <notifications@github.com>
escribió:

> Any time to work on completing this yet? Im sure some people would
> appreciate this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cordova-plugin-camera-preview/cordova-plugin-camera-preview/pull/441#issuecomment-387807699>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAhUW-R2zGvteU3FJnLJgTztelENTp8Kks5twyGsgaJpZM4SpRsX>
> .
>

westonganger commented:
Ping. Would love to get this PR merged.

manuelnaranjo commented:
@westonganger I finally updated the branch. I renamed the function to getBlob to make it more meaningful also added the docs.

My fork of the sample app also got updated github.com:manuelnaranjo/cordova-plugin-camera-preview-sample-app.git

manuelnaranjo commented:
Now that this is merged I can do it

El dom., 29 de jul. de 2018 02:23, Weston Ganger <notifications@github.com>
escribió:

> Sweet glad you could finally finish this. I would love it if you could add
> a PR to the sample app too.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cordova-plugin-camera-preview/cordova-plugin-camera-preview/pull/441#issuecomment-408643401>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAhUW1tgCG_V-tv70PaAN_RsHWvxp2JOks5uLQCcgaJpZM4SpRsX>
> .
>
